hat/assets/js/apps/Iaso/domains/pages/messages.js

```diff
@@ -14,7 +14,7 @@ const MESSAGES = defineMessages({
         defaultMessage: 'Address',
     },
     updatedAt: {
-        defaultMessage: 'Updated at',
+        defaultMessage: 'Updated',
```

Comment:
We should probably be consistent here: if we replace "Created" with "Creation" , we should replace "Updated" with "Last Update" ir something similar, otherwise it's going to look funny when we have both together